cfme/tests/cloud_infra_common/test_provisioning.py

```diff
@@ -17,11 +22,17 @@
 from cfme.utils import normalize_text
 from cfme.utils.appliance.implementations.ui import navigate_to
 from cfme.utils.blockers import BZ
+from cfme.utils.blockers import GH
 from cfme.utils.generators import random_vm_name
 from cfme.utils.log import logger
 from cfme.utils.providers import ProviderFilter
 from cfme.utils.update import update
 from cfme.utils.wait import wait_for
+logger.setLevel(logging.DEBUG)
```

Comment:
Is this a statement you added for test debug, or do you intend to set logger to debug level when this module is read?

test_modules_importable, as well as pytest collection alone, will read this module - resulting in forced debug logging level for the remainder of the process.

Reply:
I forgot to remove this. Thanks.